docs/user/04-metrics.md

```diff
@@ -418,8 +418,60 @@ spec:
 
 Note that metrics from system namespaces are excluded by default when a namespace selector for the `prometheus` or `runtime` input is not defined. However, for the `istio` and `otlp` input, metrics from system namespaces are included by default if the namespace selector is not defined.
 
+### Step 9: Enable Diagnostic Metrics
 
-### Step 9: Deploy the Pipeline
+When using the `prometheus` or `istio` input feature of the MetricPipeline, for every metric source typical scrape metrics are getting produced and these are concrete
+- up
+- scrape_duration_seconds
+- scrape_samples_scraped
+- scrape_samples_post_metric_relabeling
+- scrape_series_added
+
+These are rather technical metrics and useful for debugging and diagnose purposes. However, they are also causing data and they are by default disabled.
```

Comment:
```suggestion
These are rather technical metrics and useful for debugging and diagnostic purposes. 
```

may be we can write this whole sentence in a better way highlighting the additional space requirement.